Formula/lgeneral.rb

```diff
@@ -0,0 +1,39 @@
+class Lgeneral < Formula
+  desc "Turn-based strategy engine heavily inspired by Panzer General"
+  homepage "https://lgames.sourceforge.io/LGeneral/"
+  url "https://downloads.sourceforge.net/lgeneral/lgeneral/lgeneral-1.4.4.tar.gz"
+  sha256 "0a26b495716cdcab63b49a294ba31649bc0abe74ce0df48276e52f4a6f323a95"
+  license "GPL-2.0-or-later"
+
+  depends_on "gettext"
+  depends_on "sdl12-compat"
+  depends_on "sdl2"
+
+  def install
+    inreplace "configure", "#include <unistd.h>", "#include <sys/stat.h>\n#include <unistd.h>"
+    system "./configure", *std_configure_args,
+                         "--disable-silent-rules",
+                         "--disable-sdltest",
+                         "CFLAGS=-I/opt/homebrew/include/SDL"
```

Comment:
```suggestion
                         "CFLAGS=-I#{Formula["sdl"].opt_include}/SDL/"
```

Reply:
Reworked according to `{Formula}` solution